rex/utilities/execution.py

```diff
@@ -444,6 +457,95 @@ def scancel(job_id):
         cmd = shlex.split(cmd)
         call(cmd)
 
+    @staticmethod
+    def scancel_all():
```

Comment:
I've learned that you can specify "features" in scancel which is nice and would be a nice feature to add here,
for example you can do `scancel -p short` to cancel all short jobs. It would also be nice to be able to cancel a list of jobs. 

Could we add another function that would let you:
```
scancel(job_ids=None, features=None)
```
where job_ids could be a single job id, or list of job ids, and features could be like `-p short`

Reply:
Agreed. scancel will still take in "job_id" but that job id can now be a list or a string.